osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs

```diff
@@ -244,7 +244,13 @@ protected override void Update()
                 else if (slidingSample.IsPlaying)
                     slidingSample.Stop();
             }
+        }
+
+        protected override void UpdateAfterChildren()
+        {
+            base.UpdateAfterChildren();
 
+            // It's important that this is done in UpdateAfterChildren so that the SliderBody has a chance to update its Size and PathOffset on Update.
```

Comment:
These kinds of changes always scare me, because I'm never sure if this is going to be changing the delay between input handling / positioning. ie. this is responsible for changing the slider ball's position, which also controls the hit area of the follow circle.

I think this is okay though.

Reply:
Not sure if it's just me but I feel like there's a half of an explanation missing here. I don't understand what this change has to do with the rest of the diff.

Reply:
Yeah that too. The inline comment doesn't do much to explain the *actual* hidden reason, so needs improvement.

Reply:
I've added clarification that this is because `PlaySliderBody` schedules a refresh once on Update whenever the slider path gets edited.